mono/mini/mini.c

```diff
@@ -3363,7 +3363,9 @@ mini_method_compile (MonoMethod *method, guint32 opts, MonoDomain *domain, JitFl
 		for (i = 0; verbose_method_names [i] != NULL; i++){
 			const char *name = verbose_method_names [i];
 
-			if ((strchr (name, '.') > name) || strchr (name, ':')) {
+			if (name && *name == '*') {
+				cfg->verbose_level = 4;
```

Comment:
Simply running with -v -v -v -v will achieve the same effect.

Reply:
@vargaz ok I've just removed it. What about separators?